--- di-4.23/Makefile~	2010-05-11 07:55:01.000000000 +0200
+++ di-4.23/Makefile	2010-05-13 09:29:39.486116575 +0200
@@ -137,9 +137,9 @@
 	copy /y NUL: reqlibs.txt
 	$(MAKE) \
 		CC=cl LD=cl EXE_EXT=".exe" OBJ_EXT=".obj" \
-		CFLAGS="$(CFLAGS) -nologo -O2 \
+		CFLAGS="$(CFLAGS) $(OPTFLAGS) -nologo" \
 		-DDEFAULT_FORMAT=\\\"MbuvpT\\\"" \
-		LDFLAGS="-nologo -O2" di-programs
+		LDFLAGS="-nologo" di-programs
 
 # This was tested using cygwin
 windows-gcc:
@@ -162,8 +162,7 @@
 		CC=mingw32-gcc LD=mingw32-gcc \
 		EXE_EXT=".exe" OBJ_EXT=".o" \
 		CFLAGS="$(CFLAGS) -g -O2 \
-		-DDEFAULT_FORMAT=\\\"MbuvpT\\\"" \
-		LDFLAGS="-g -O2" mingw-$(PROG).exe
+		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\""
 
 os2-gcc:
 	@echo ':' > di.env;chmod a+rx di.env
