--- di-4.20/Makefile~	2010-03-31 17:30:14.000000000 +0200
+++ di-4.20/Makefile	2010-04-02 16:24:55.767775880 +0200
@@ -138,7 +138,7 @@
 		CC=cl LD=cl EXE_EXT=".exe" OBJ_EXT=".obj" \
 		CFLAGS="$(CFLAGS) -nologo -O2 \
 		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-nologo -O2" di-programs
+		LDFLAGS="-nologo" di-programs
 
 # This was tested using cygwin
 windows-gcc:
@@ -148,8 +148,7 @@
 	$(MAKE) \
 		CC=gcc LD=gcc EXE_EXT=".exe" OBJ_EXT=".o" \
 		CFLAGS="$(CFLAGS) -g -O2 \
-		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-g -O2" di-programs
+		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\""
 
 os2-gcc:
 	@echo ':' > di.env;chmod a+rx di.env
@@ -279,12 +278,12 @@
 	@if [ -f reqlibs.txt ];then \
 		echo "*** Using libs: `$(CAT) reqlibs.txt`"; fi
 	$(CC) -o $(PROG)$(EXE_EXT) \
-		$(CFLAGS) $(LDFLAGS) $(OBJECTS) $(LIBS) \
+		$(CFLAGS) $(OPTFLAGS) $(LDFLAGS) $(OBJECTS) $(LIBS) \
 		`if [ -f reqlibs.txt ];then $(CAT) reqlibs.txt; fi`
 
 hasnls$(EXE_EXT):	hasnls$(OBJ_EXT)
 	$(CC) -o hasnls$(EXE_EXT) \
-		$(CFLAGS) $(LDFLAGS) \
+		$(CFLAGS) $(OPTFLAGS) $(LDFLAGS) \
 		hasnls$(OBJ_EXT)
 
 ###
