--- di-4.22/Makefile~	2010-05-06 06:50:33.000000000 +0200
+++ di-4.22/Makefile	2010-05-06 12:22:40.266987097 +0200
@@ -136,9 +136,9 @@
 	$(CP) -f config.win config.h
 	$(MAKE) \
 		CC=cl LD=cl EXE_EXT=".exe" OBJ_EXT=".obj" \
-		CFLAGS="$(CFLAGS) -nologo -O2 \
+		CFLAGS="$(CFLAGS) $(OPTFLAGS) -nologo" \
 		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-nologo -O2" di-programs
+		LDFLAGS="-nologo" di-programs
 
 # This was tested using cygwin
 windows-gcc:
@@ -148,8 +148,7 @@
 	$(MAKE) \
 		CC=gcc LD=gcc EXE_EXT=".exe" OBJ_EXT=".o" \
 		CFLAGS="$(CFLAGS) -g -O2 \
-		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-g -O2" di-programs
+		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\""
 
 os2-gcc:
 	@echo ':' > di.env;chmod a+rx di.env
