--- di-4.12/Makefile~	2008-01-21 08:32:54.000000000 +0100
+++ di-4.12/Makefile	2008-01-21 08:32:44.000000000 +0100
@@ -70,7 +70,7 @@
 
 # have to get various environment variables set up.
 all:
-	./Build
+	./Build -mkc
 
 di-all:	config.h $(PROG)$(EXE_EXT) hasnls$(EXE_EXT)
 
@@ -93,15 +93,14 @@
 		CC=cl LD=cl EXE_EXT=".exe" OBJ_EXT=".obj" \
 		CFLAGS="$(CFLAGS) -nologo -O2 \
 		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-nologo -O2" $(PROG).exe
+		LDFLAGS="-nologo" $(PROG).exe
 
 # This was tested using cygwin
 windows-gcc:	wconfig.h
 	$(MAKE) -f Makefile \
 		CC=gcc LD=gcc EXE_EXT=".exe" OBJ_EXT=".o" \
 		CFLAGS="$(CFLAGS) -g -O2 \
-		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\"" \
-		LDFLAGS="-g -O2" $(PROG).exe
+		-DDEFAULT_FORMAT=\\\"sMbuvpT\\\""
 
 ####################
 
@@ -202,14 +201,14 @@
 		getopt$(OBJ_EXT)
 	@test ! -f reqlibs.txt || echo "*** Using libs: `cat reqlibs.txt`"
 	$(CC) -o $(PROG)$(EXE_EXT) \
-		$(CFLAGS) $(LDFLAGS) \
+		$(CFLAGS) $(OPTFLAGS) $(LDFLAGS) \
 		di$(OBJ_EXT) dilib$(OBJ_EXT) \
 		getopt$(OBJ_EXT) $(LIBS) \
 		`test ! -f reqlibs.txt || cat reqlibs.txt`
 
 hasnls$(EXE_EXT):	hasnls$(OBJ_EXT)
 	$(CC) -o hasnls$(EXE_EXT) \
-		$(CFLAGS) $(LDFLAGS) \
+		$(CFLAGS) $(OPTFLAGS) $(LDFLAGS) \
 		hasnls$(OBJ_EXT)
 
 ###
