--- di-4.21/Makefile~	2010-04-21 09:52:17.420464903 +0200
+++ di-4.21/Makefile	2010-04-21 09:54:07.989625502 +0200
@@ -229,7 +229,7 @@
 	-$(TEST) -f $(TARGET) && $(MV) -f $(TARGET) $(TARGET).old
 	-$(TEST) -f $(MTARGET) && $(MV) -f $(MTARGET) $(MTARGET).old
 	$(CP) -f $(PROG)$(EXE_EXT) $(TARGET)
-	-$(LN) -s $(PROG)$(EXE_EXT) $(MTARGET)
+	-$(LN) -s /usr/bin/di $(MTARGET)
 	-$(RM) -f $(TARGET).old $(MTARGET).old > /dev/null 2>&1
 	@-./hasnls$(EXE_EXT) && echo "# NLS enabled"
 	@-./hasnls$(EXE_EXT) || echo "# NLS not enabled"
